loki/sourcefile.py

```diff
@@ -424,10 +424,10 @@ def make_complete(self, **frontend_args):
     def source(self):
         return self._source
 
-    def to_fortran(self, conservative=False, cuf=False):
+    def to_fortran(self, conservative=False, cuf=False, style=None):
         if cuf:
-            return cufgen(self, conservative=conservative)
-        return fgen(self, conservative=conservative)
+            return cufgen(self, conservative=conservative, style=style)
```

Comment:
`cufgen` via this entry point is currently not covered.

Reply:
This is actually quite awkward to test, as it's merely a convenience entry point. I think it's safe to leave this until we generalised the backend dispatch a bit more (see above comment).